recipe/meta.yaml

```diff
@@ -19,18 +19,18 @@ build:
         - vc14  # [win and py35]
 
 build:
-    number: 1
+    number: 2
 
 requirements:
     build:
         - python  # [win]
         - cmake  # [win]
         - zlib
-        - jpeg
+        - jpeg 8d
```

Comment:
> Apologies. Happy to move to the latest version so long as it is pinned *correctly* 😄 

I am still navigating down the stream to pin and re-build these. (Although I passed here and missed it.)

I was just afraid that you had a reason to use `8d` instead of `9b`.